doc/userguide/lua/libs/ja3.rst

```diff
@@ -0,0 +1,87 @@
+JA3
+---
+
+JA3 details are exposes to Lua scripts with the
+``suricata.ja3`` library, For example::
+
+  local ja3 = require("suricata.ja3")
+
+If you want to use ja3, you can either set suricata.yaml option
+``app-layer.protocols.tls.ja3-fingerprints`` to true,
+or specify it in the ``init`` function of your lua script
+by calling ``ja3.enable_ja3()``::
+
+  function init (args)
+    ja3.enable_ja3()
+    return {}
+  end
+
+For use in rule matching, the rule may **hook** into a TLS or QUIC
+transaction state if you want to match on only one of these protocols.
+Or you should use need ``ja3`` or ``ja3s`` in your init script::
+
+  function init (args)
+    ja3.enable_ja3()
+    local needs = {}
+    needs["ja3s"] = tostring(true)
```

Comment:
I think we should prefer `needs["ja3s"] = true` now that it is supported.

Reply:
cool